Gordon360/Services/RecIM/ActivityService.cs

```diff
@@ -26,15 +26,15 @@ public IEnumerable<LookupViewModel> GetActivityLookup(string type)
             switch (type)
             {
                 case "status":
-                    return _context.ActivityStatus
+                    return _context.ActivityStatus.Where(query => query.ID != 0)
```

Comment:
All deleted ID's are 0, this status should never be returned to the UI when they attempt a lookup